napari/layers/points/_tests/test_points_mouse_bindings.py

```diff
@@ -0,0 +1,359 @@
+import numpy as np
+import collections
+from napari.layers import Points
+from napari.utils.interactions import (
+    ReadOnlyWrapper,
+    mouse_press_callbacks,
+    mouse_move_callbacks,
+    mouse_release_callbacks,
+)
+
+
+def create_known_points_layer():
```

Comment:
Probably this should be a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html)?

Reply:
done